src/routes/ComparisonsRouter.ts

```diff
@@ -1,20 +1,29 @@
 import express from 'express';
 import pg from '../database/knex';
 import { Comparison } from '@src/models/Comparisons';
+import ComparisonsService from '@src/services/ComparisonsService';
 
 
 const comparisonRouter = express.Router();
 
 // Define routes
 comparisonRouter.get('/', async function (req, res) {
-  const comparisons: Comparison[] = await pg('comparisons').select('*');
+  const comparisons: Comparison[] = await ComparisonsService.getAllComparisons();
   res.send(comparisons);
 });
 
 comparisonRouter.get('/:id', async function (req, res) {
```

Comment:
Would the UI ever call this endpoint? Idk the case when a client has a comparison ID and wants to look up just one comparison.

I imagined that the client for each new comparison just calls /comparison and it returns a random comparison between two subjects. That call alone should have all the information we need to display the comparison right?

Reply:
Will remove this.